### PR TITLE
OSDOCS-2845: rosa docs list --version argument for rosa create-accounts that has been removed

### PR DESCRIPTION
**Pull request by nermina-redhat**
For dedicated-4

https://issues.redhat.com/browse/OSDOCS-2845

😎 Browse the preview: https://deploy-preview-38412--osdocs.netlify.app

### CODE REVIEW

**Comment by nermina-redhat** [comment]
@okashi18, please review

File: modules/rosa-upgrading-preparing-4-7-to-4-8.adoc [hunk 1]
```diff
@@ -24,7 +24,7 @@ You must meet the following requirements before upgrading a {product-title} (ROS
 +
 [source,terminal]
 ----
-$ rosa create account-roles --mode auto --version 4.8
```
Comment (paulczar) [inline modules/rosa-upgrading-preparing-4-7-to-4-8.adoc:27]: we probably don't need to remove the `--mode auto` ?  most people will likely want to do auto mode.

**Comment by nermina-redhat** [comment]
@Paul Czarkowski ***@***.***> this is what the change looks like:
https://deploy-preview-38412--osdocs.netlify.app/openshift-rosa/latest/rosa_getting_started_sts/rosa_creating_a_cluster_with_sts/rosa-sts-creating-a-cluster-quickly.html

**Comment by bmcelvee** [comment]
Looks like this one needs the `--mode auto` flag re-added, and commits squashed. Otherwise, LGTM!

**Pull request closed without merging**